modules/cli/src/main/java/org/apache/ignite/internal/cli/commands/node/NodeReplCommand.java

```diff
@@ -17,15 +17,23 @@
 
 package org.apache.ignite.internal.cli.commands.node;
 
+import org.apache.ignite.internal.cli.commands.BaseCommand;
 import org.apache.ignite.internal.cli.commands.node.config.NodeConfigReplCommand;
 import org.apache.ignite.internal.cli.commands.node.metric.NodeMetricReplCommand;
 import org.apache.ignite.internal.cli.commands.node.status.NodeStatusReplCommand;
 import org.apache.ignite.internal.cli.commands.node.version.NodeVersionReplCommand;
 import picocli.CommandLine.Command;
+import picocli.CommandLine.HelpCommand;
 
 /** Node command in REPL mode. */
 @Command(name = "node",
-        subcommands = {NodeConfigReplCommand.class, NodeStatusReplCommand.class, NodeVersionReplCommand.class, NodeMetricReplCommand.class},
+        subcommands = {
+                NodeConfigReplCommand.class,
+                NodeStatusReplCommand.class,
+                NodeVersionReplCommand.class,
+                NodeMetricReplCommand.class,
+                HelpCommand.class
+        },
         description = "Node operations")
-public class NodeReplCommand {
+public class NodeReplCommand extends BaseCommand {
```

Comment:
What is this inheritance for?

Reply:
All commands must support '--help' option, if want to use 'command help' syntax.